iwyu_include_picker.h

```diff
@@ -89,14 +89,13 @@ class IncludePicker {
   // Indicate that the given quoted include should be considered
   // a "private" include.  If possible, we use the include-picker
   // mappings to map such includes to public (not-private) includs.
-  void MarkIncludeAsPrivate(const string& quoted_include);
+  void MarkIncludeAsPrivate(const string& quoted_filepath_pattern);
 
   // Add this to say that "any file whose name matches the
   // friend_regex is allowed to include includee_filepath".  The regex
   // uses the POSIX Entended Regular Expression syntax and should
   // match a quoted-include (starting and ending with "" or <>).
-  void AddFriendRegex(const string& includee_filepath,
-                      const string& quoted_friend_regex);
+  void AddFriendRegex(const string& includee, const string& friend_regex);
```

Comment:
Here I would change argument names in .cc file to be the same as in .h as names in .h are more specific.